src/config.ml

```diff
@@ -140,10 +170,22 @@ let load_user_config_file () =
     default
 
 let adapt_display config ~output_is_a_tty =
-  if config.display = Progress &&
-     not output_is_a_tty &&
-     not inside_emacs
-  then
-    { config with display = Quiet }
-  else
+  (* Progress isn't meaningful if inside a terminal (or emacs),
+   * so reset the display to Quiet if the output is getting 
+   * piped to a file or something. *)
+  let config = if config.display = Progress &&
```

Comment:
I believe the old formatting was preferable.

Reply:
Done